MINOR: Add option to rebuild source for system tests

abbccdda opened the pull request:
Save developer time for running mandatory commands.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

Commit:
add option to rebuild source for system tests

abbccdda commented:
Retest this please

abbccdda commented:
Retest this please

abbccdda commented:
@mjsax @hachikuji Could you take a look when you got time?

abbccdda commented:
@mjsax Could we get this merged?

hachikuji commented on tests/docker/run_tests.sh:
```diff
@@ -24,6 +24,11 @@ die() {
 
+read -p "Rebuild source? (y/n) " rebuild
+if [ "$rebuild" == "y" ]; then
+    ./gradlew clean systemTestLibs
```
Not sure it's safe to depend of the local directory. Perhaps it should be relative to `$SCRIPT_DIR`?

I am also not too sure a prompt is the best way. If we have any automated systems using this script, it would fail, right? Could we pass an optional flag instead? Something like `--rebuild` which defaults to false?

abbccdda commented:
@hachikuji Addressed comment. Note that it's not required to change directory because the script has to be run under Kafka root when triggered by human user.

abbccdda commented:
Retest this please

abbccdda commented:
Retest this please

abbccdda commented:
@mjsax Could we merge this?

mjsax commented:
@hachikuji if you are happy with this PR, feel free to merge.

abbccdda commented:
@hachikuji WDYT?
